src/views/LoginView.vue

```diff
@@ -70,17 +137,16 @@ onBeforeMount(() => {
   }
 });
 
-const ldapLogin = async (event: Event) => {
-  event.preventDefault();
-  if (!values.username || !values.password) return;
-  await authStore.gewisLdapLogin(values.username, values.password, apiService).then(() => {
-    console.error("loggedin");
+const ldapLogin = loginForm.handleSubmit(async () => {
+  if (!username.value.modelValue || !password.value.modelValue) return;
+  await authStore.gewisLdapLogin(username.value.modelValue,  password.value.modelValue, apiService).then(() => {
     if (authStore.getUser) userStore.fetchCurrentUserBalance(authStore.getUser.id, apiService);
     router.push({ name: 'home' });
   }).catch((error) => {
     console.error(error);
   });
-};
+});
+
 const loginViaGEWIS = () => {
```

Comment:
Issue maken / taggen dat inloggen via GEWIS nog niet werkt? we doen nog niks met de callback.

Reply:
Hier idd een todo maken die #32 benoemd.

Reply:
added TODO and issue # in commit https://github.com/GEWIS/sudosos-frontend-vue3/pull/31/commits/c0cd5afae61160be4c481f76a38752281f930aa6